libcodechecker/libhandlers/cmd.py

```diff
@@ -607,9 +607,8 @@ def add_arguments_to_parser(parser):
     sum_p = subcommands.add_parser(
         'sum',
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-        description="Show the count of checker reports per checker for some "
-                    "analysis runs.",
-        help="Show number of reports per checker.")
+        description="Show checker statistics for some analysis runs.",
```

Comment:
As the description of this command was changed, the `docs/user_guide.md` file should also be updated to reflect the new help.